HyperPlay-Gaming/hyperplay-desktop-client
# [FIX] Check if game is available on game launch

Fix a regression where we stopped checking if the game is available before launching.
In case the executable or the game folder was deleted or moved it will show a message using the native dialog (necessary since it might be launched without GUI).

It shows an option to remove the game from the installed list as well:
![imagem](https://github.com/HyperPlay-Gaming/hyperplay-desktop-client/assets/26871415/31d94bd9-b51b-4ae3-83ad-abe609db39bc)

## How to test

1. Rename a currently installed game folder
2. Launch the game from the game card or game page.
3. The dialog should show after the onboarding dialog.

---

Use the following Checklist if you have changed something on the Backend or Frontend:
